doc/mediasdk-man.md

```diff
@@ -7564,6 +7564,37 @@ This structure is used to pass decryption status report index for Common Encrypt
 
 This structure is available since SDK API 1.30.
 
+## <a id='mfxExtPartialBitstreamParam'>mfxExtPartialBitstreamParam</a>
+
+**Definition**
+
+```C
+typedef struct {
+    mfxExtBuffer    Header;
+    mfxU32          BlockSize;        /* output block granulatiry for Granularity = MFX_PARTIAL_BITSTREAM_BLOCK */
+    mfxU16          Granularity;      /* granulatiry of the partial bitstream: slice/block/any */
+    mfxU16          reserved[8];
+} mfxExtPartialBitstreamParam;
+```
+
+**Description**
+
+This structure is used by AVC HW encoder to output parts of bitstream as soon as they ready. The application can attach this extended buffer to the [mfxFrameSurface1](#mfxFrameSurface) structure at init time.
+If this option is turned ON and session configured in single thread, then AVC encoder can output bitstream by part based with required granularity. <br>
+<br>This parameter is valid only during initialization and reset. Absence of this buffer means default bitstream output behavior.<br>
```

Comment:
During initialization and reset I have mfxVideoParam and I don't have mfxFrameSurface1...

Reply:
See comment before, you right